testcafe/tests/helpers/utils.js

```diff
@@ -92,11 +92,22 @@ export const getLastExecutedCommand = ClientFunction(
 )
 
 //@param{string} filepath : Expected full path to file
-export async function checkLocalFile(filepath) {
-  await t.wait(data.DOWNLOAD_DELAY)
-  await t.expect(fs.existsSync(filepath)).ok(`${filepath} doesn't exist`)
-  logger.debug(`${filepath} exists on local drive`)
+export async function checkLocalFile(t, filepath) {
+  for (var i = 0; i < data.DOWNLOAD_DELAY; i++) {
+    await t.wait(10)
+    if (fs.existsSync(filepath)) {
+      logger.info(
+        `${filepath} exists on local drive. Waited for a total of ${i.toString()} microseconds`
+      )
+      return
```

Comment:
Why `return`? Did you mean `break`?

Reply:
Changed to `break`

Reply:
Alright, let's see if the tests still pass after this change, since before they weren't running at all :/

Reply:
~~return it's the same as break in a for loop, no?~~

Oh no. You right! OMG